frontend/src/theme/primitives/typography.ts

```diff
@@ -16,9 +16,15 @@
  */
 
 export const fonts: { [key: string]: string } = {
-  sansSerif: "IBM Plex Sans, sans-serif",
-  monospace: "IBM Plex Mono, monospace",
-  serif: "IBM Plex Serif, serif",
+  sansSerif:
+    '"Source Sans Pro", -apple-system, BlinkMacSystemFont, "Segoe UI", Helvetica, ' +
+    "Arial, sans-serif",
+  monospace:
+    '"Source Code Pro", iawriter-mono, Nitti, Menlo, Courier, monospace',
+  serif:
+    '"Source Serif Pro", Lyon-Text, Georgia, YuMincho, "Yu Mincho", "Hiragino Mincho ProN", ' +
+    '"Hiragino Mincho Pro", "Songti TC", "Songti SC", SimSun, ' +
+    '"Nanum Myeongjo", NanumMyeongjo, Batang, serif',
```

Comment:
I just feel this is necessary...why all these font choices? They are quite...extensive

Reply:
Now that we're shipping our own fonts, this is less needed. Reverted.